roles/elastic-stack/ansible-kibana/tasks/main.yml

```diff
@@ -134,6 +134,31 @@
   when:
     - not build_from_sources
 
+- name: Wait for Elasticsearch port
+  wait_for: host={{ elasticsearch_network_host }} port={{ elasticsearch_http_port }}
+
+- name: Select correct API protocol
+  set_fact:
+    elastic_api_protocol: "{% if kibana_xpack_security %}https{% else %}http{% endif %}"
+  
+- name: Attempting to delete legacy Wazuh index if exists
+  uri:
+    url: "{{ elastic_api_protocol }}://{{ elasticsearch_network_host }}:{{ elasticsearch_http_port }}/.wazuh"
+    method: DELETE
+    user: "{{ elasticsearch_xpack_security_user }}"
+    password: "{{ elasticsearch_xpack_security_password }}"
+    validate_certs: no
+    status_code: 200, 404
+  ignore_errors: yes
```

Comment:
Adding a comment to notify users that this curl can be skipped because the app does have a default config would be great IMO

Reply:
Fixed in `71e52de`

Now it doesn't ignore errors on index removal